dependencyManagement/build.gradle.kts

```diff
@@ -27,24 +27,24 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val TEST_SNAPSHOTS = rootProject.findProperty("testUpstreamSnapshots") == "true"
 
 // This is the only version that affects the released artifact.
-val otelVersion = "1.18.0"
-val otelSnapshotVersion = "1.19.0"
+val otelVersion = "1.19.0"
+val otelSnapshotVersion = "1.20.0"
 
 // All versions below are only used in testing and do not affect the released artifact.
 
 val DEPENDENCY_BOMS = listOf(
-  "com.amazonaws:aws-java-sdk-bom:1.12.321",
-  "com.fasterxml.jackson:jackson-bom:2.13.4",
+  "com.amazonaws:aws-java-sdk-bom:1.12.322",
+  "com.fasterxml.jackson:jackson-bom:2.14.0-rc2",
```

Comment:
Should this be using an `rc`?

Reply:
It was in the summary: Updating jacksom-bom to 2.14.0-rc2 because it is only used in tests.

Reply:
Reading is hard!